### DISCUSSION TITLE
mac 和 Chrome 无法使用GPT3.5 或GPT4 问题求大佬解答🙏

### DISCUSSION THREAD

**Discussion by SadirKhan01** [nextai-translator/nextai-translator#1381] [Q&A]
首先非常感谢大佬分享这么好的软件！！

我有一年时间没有用这个软件了，今天打开突然发现用不了了，提示如下：

You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.

于是我重新申请了API,问题依旧没有改变，mac 端 Chrome 插件都是无法使用，我觉得可能是我账户出了问题……

所以从朋友借来了gpt 4 的api来尝试，提示如下：

The model `gpt-4` does not exist or you do not have access to it. Learn more: https://help.openai.com/en/articles/7102672-how-can-i-access-gpt-4.

现在我不知道怎么解决这个问题，求大佬帮忙分析原因…

我真的很需要这个平台

拜托了🙏🙏🙏

**Comment by yetone** [comment]
你的 OpenAI API 没有额度了，请充值
